.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
+}
```

Comment:
When is this file used? Some cake task?

Reply:
It’s used only if you want to manually test `coffee --transpile`. So if you have the repo checked out and you want to run `coffee --transpile --eval "import path from 'path'; console.log path.sep"` and have it just work, without needing to create `.babelrc` first. I could just as easily put this into a `package.json` `babel` key, or leave it out. It’s not used by any `cake` tasks or any other part of the normal workflow. I just thought it would make development more convenient.

Reply:
I see, good idea!